### PR TITLE
Demo hotfixes

### PR DESCRIPTION
**Pull request by cbuchko**
Change some small things for the demo. 

- Professor names are now displaying
- Changed the query to cache-and-network
- Added a loader for when query grabs from network

### CODE REVIEW

File: src/components/Schedule/CalendarView.tsx [hunk 1]
```diff
@@ -51,9 +51,7 @@ const splitCourseDays = (course: CourseSection) => {
 //convert Course object data from backend to the structure DevExtreme expects
 const buildAppointments = (courses: CourseSection[]) => {
   const appointments = courses.flatMap((course) => {
-    // const professors = course.professors.map((prof) => prof.displayName);
-    const professors = ["Joe Biden"];
```
Comment (jayden-chan) [inline src/components/Schedule/CalendarView.tsx:55]: RIP Joe Biden

File: src/pages/Schedule.tsx [hunk 2]
```diff
@@ -51,18 +58,14 @@ export const Schedule = () => {
     data: baseScheduleData,
     loading: scheduleLoading,
     error: scheduleError,
-  } = useQuery(COURSES, {});
+  } = useQuery(COURSES, { fetchPolicy: "cache-and-network" });
```
Comment (jayden-chan) [inline src/pages/Schedule.tsx:61]: Does this actually fix the issue? I figured you would have used `network-only`

**Review by jayden-chan — APPROVED** [review]

File: src/pages/Schedule.tsx (hunk 2 above)
Comment (jayden-chan) [reply src/pages/Schedule.tsx:61]: Oh yeah nvm I think this does do what we want :+1:

Comment (cbuchko) [reply src/pages/Schedule.tsx:61]: yeah this works fine, I thought doing cache-and-network would help for querying when it doesn't need to refetch, but it looks like it does the full network query everytime anyways lmao

**Review by Adamk8 — APPROVED** [review]
Before you merge, on line 133 of components/SurveyCourseList can you change FALL to SUMMER

**Pull request merged**